### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (#sidedocsframe;a[href="\#doc\:javascript\/interfaces"])

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2280]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/#editor)
#### Element path
#sidedocsframe;a[href="\#doc\:javascript\/interfaces"]
#### Snippet
`<a rel="nofollow noopener" href="#doc:javascript/interfaces">Interfaces</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.98 (foreground color: #4183c4, background color: #ffffff, font size: 12.0pt, font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

links in side docs

![Screen Shot 2019-07-17 at 5 53 30 PM](https://user-images.githubusercontent.com/5615930/61421410-d8227b80-a8bb-11e9-89f9-44ac1a87201a.png)
